### PR TITLE
 Add dubbo basic usage and dubbo-provider configuration #86,Baiji269_Team 5

### PR DESCRIPTION
**Pull request by YanW1222**
## What is the purpose of the change
增加Dubbo基本用法-Dubbo Provider配置的英文版，并且修改了5个中文版中的错误：
1.配置项详解 第2点 zookeeper地质—->地址（两处）
2.映射规则 第三个例子 端口号不一致——>1099改1234
3.删掉句尾多余的2
4.支持的主要属性——>纠正list格式问题

## Brief changelog

add dubbo basic usage-dubbo provider configuration English version.

Follow this checklist to help us incorporate your contribution quickly and easily:

- [x] Make sure there is a [GITHUB_issue](https://github.com/apache/incubator-dubbo-website/issues) filed for the change (usually before you start working on it). Trivial changes like typos do not require a GITHUB issue. Your pull request should address just this issue, without pulling in other changes - one PR resolves one issue.
- [x] Format the pull request title like `Fix UnknownException when host config not exist #XXX`. Each commit in the pull request should have a meaningful subject line and body.
- [x] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
- [x] Test your code locally by running `docsite start`, and make sure it works as expected.
- [x] Make sure no files under build directory is added.
- [x] If this contribution is large, please follow the [Software Donation Guide](https://github.com/apache/incubator-dubbo/wiki/Software-donation-guide).

### CODE REVIEW

**Comment by liaoandi** [comment]
Comment from Team 9

1.  line 6
> This chapter mainly talking about how to configure dubbo
"talking about" should be “talks about”.

2. line 6
> it can be divided into the following mode
"mode" should be "modes".

3. line 92
> can be configured corresponding to method
"method" should be "the method".

4. line 107
> If you have any questions about consumer
"questions" should be "question".

5. line 131 to 136
> * For example: dubbo.application.name=foo equivalents to <dubbo:application name="foo" />
* For example: dubbo.registry.address=10.20.153.10:9090 equivalents to <dubbo:registry address="10.20.153.10:9090" />
 If there are multiple configurations having the same tag name, they can be distinguished by id, and if there is no id, the configurations will be applied to all tags with the same name.
* For example: dubbo.protocol.rmi.port=1234 equivalents to <dubbo:protocol id="rmi" name="rmi" port="1234" /> 
* For example: dubbo.registry.china.address=10.20.153.10:9090 equivalents to <dubbo:registry id="china" address="10.20.153.10:9090" />

"equivalents to" should be "is equivalent to" or" equals to".

6. line 146
>  which may change to throwing exceptions later.
"be changed to"  will be better.

7. line 243
> and lets explain the complete use of Dubbo provider by going through a complete example.
"lets" should be "let's, and "usage" can take place of "use" here.

**Comment by YanW1222** [comment]
Some thoughts about the guide from team_5:
1. We learned how to configure dubbo, we can use 4 different ways to configure it, each of which have their own advantages and disadvantages. We need to decide which one to use according to the situation.
2.This chapter talks about the relationship between each configuration which really deserve to learn.
3.It's a great start-up document for new learners.

File: site_config/blog.js [hunk 1]
```diff
@@ -72,6 +72,13 @@ export default {
         dateStr: 'August 7th，2018',
         desc: 'Your First Dubbo Demo',
         link: '/en-us/blog/dubbo-101.html',
+      },
+      {
+        title: 'Dubbo Basic Usage Dubbo Provider Configuration',
+        author: 'Yan Wei and Ziyang Wu',
+        dateStr: 'August 14th，2018',
+        desc: 'Dubbo Basic Usage Dubbo Provider Configuration',
+        link: '/en-us/blog/dubbo-basic-usage-dubbo-provider-conofiguration.html',
```
Comment (diecui1202) [inline site_config/blog.js]: please fix the file name typo.

**Comment by diecui1202** [comment]
The same to #104, which has been merged. So this PR will be closed.

**Pull request closed without merging**